source/opt/ir_context.cpp

```diff
@@ -365,6 +373,49 @@ void IRContext::KillNamesAndDecorates(Instruction* inst) {
   KillNamesAndDecorates(rId);
 }
 
+void IRContext::KillOperandFromDebugInstructions(Instruction* inst) {
```

Comment:
You should have unit tests for this function in https://github.com/KhronosGroup/SPIRV-Tools/blob/master/test/opt/ir_context_test.cpp